qcodes/dist/tests/station/example.station.yaml

```diff
@@ -0,0 +1,59 @@
+instruments:
```

Comment:
[the station notebook also contains an example with comments and stuff](https://nbviewer.jupyter.org/github/QCoDeS/Qcodes/blob/f24eb33b6a078cc8d8d408168dc809fbe7170128/docs/examples/Station.ipynb#Example-of-YAML-Station-configuration) - to avoid duplication, would it make sense to move that example into this file, and render this file in the notebook?

Reply:
I have not found a way how to do this nicely. And as divergence is not of an big issue here, I'd suggest: let's leave it for now.